image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```diff
@@ -139,6 +139,10 @@ collector:
     enabled: null # bool
   runtimeConfig:
     enabled: null # bool
+    networking:
+      externalIps:
+        enable: null # bool
```

Comment:
For consistency:
```suggestion
        enabled: null # bool
```

Reply:
This would need to be changed in the protobuf definitions and collector. Collector expects the ConfigMap to have `enable`, not `enabled`. `enable` when into 4.6 as tech preview, so it can still be changed before becoming GA, but I am not a fan of the idea.

Reply:
I don't mean the internal name of the setting in the configmap or protos. I mean the user-visible helm chart setting. Unless I'm missing something, these two can differ.

Reply:
Ideally the ConfigMap would be read into collector using code from the protobuf library and the code generated from the protobuf definitions, such that if new fields are added to the protobuf definition for collector runtime configuration no changes would have to be made to the collector code. In that case there would have to be agreement between the protobuf definitions and the ConfigMap. I also think there should be agreement between the ConfigMap and helm charts. Currently if a new field is added to the runtime configuration the code for reading the ConfigMap in collector will have to be changed to include that field. I am hoping to change that for 4.7.

Reply:
> I also think there should be agreement between the ConfigMap and helm charts.

Here (and only here) I disagree. We should strive for a streamlined experience for the helm chart user (i.e. enabling features should **not** be done using an `enabled` key in most places but `enable` key in some other places). The user should be shielded from the ConfigMap, which is an implementation detail.

Reply:
I have changed `enable` to `enabled` in the helm charts. I will change the protobuf definition later.